src/twisted/internet/defer.py

```diff
@@ -159,6 +165,65 @@ def maybeDeferred(f, *args, **kw):
         return succeed(result)
 
 
+def unbox(deferred, raises=True, passthrough=True):
+    """
+    Unbox a L{Deferred} that has already been fully processed.
+
+    Take an already processed (i.e. no longer waiting on any callbacks to be
+    fired) and unbox it. If the result is a L{Failure}, then it will re-raise
```

Comment:
Missing a word here (deferred), and I think it'd be a bit nicer to flip the order of the english sentence so that you talk about the happy normal case first, and then "it reraises" is your otherwise at the end.